plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk

```diff
@@ -491,15 +491,15 @@ group edgeRouting {
         description 
             "Alter the distribution of the loops around the node. It only takes effect for PortConstraints.FREE."
         default = SelfLoopDistributionStrategy.NORTH
-        targets parents
+        targets nodes
     }
     
     option selfLoopOrdering: SelfLoopOrderingStrategy {
         label "Self-Loop Ordering"
          description "Alter the ordering of the loops they can either be stacked or sequenced.
                 It only takes effect for PortConstraints.FREE."
         default = SelfLoopOrderingStrategy.STACKED
-        targets parents
+        targets nodes
```

Comment:
Hence this works for hierarchical nodes and atomic nodes, right?

Reply:
Exactly.